src/minoversikt/minoversikt-listehode.tsx

```diff
@@ -166,6 +166,36 @@ function MinOversiktListeHode({
                     skalVises={valgteKolonner.includes(Kolonne.TOLKEBEHOV_SIST_OPPDATERT)}
                 />
 
+                <SorteringHeader
+                    sortering={Sorteringsfelt.BOSTED_KOMMUNE}
+                    onClick={sorteringOnClick}
+                    rekkefolge={sorteringsrekkefolge}
+                    erValgt={sorteringsfelt === Sorteringsfelt.BOSTED_KOMMUNE}
+                    tekst="Bosted"
+                    className="col col-xs-2"
+                    headerId="bosted_kommune"
+                    skalVises={valgteKolonner.includes(Kolonne.BOSTED_KOMMUNE)}
+                />
+                <SorteringHeader
+                    sortering={Sorteringsfelt.BOSTED_BYDEL}
+                    onClick={sorteringOnClick}
+                    rekkefolge={sorteringsrekkefolge}
+                    erValgt={sorteringsfelt === Sorteringsfelt.BOSTED_BYDEL}
+                    tekst="Bosted detaljer"
+                    className="col col-xs-2"
+                    headerId="bosted_bydel"
+                    skalVises={valgteKolonner.includes(Kolonne.BOSTED_BYDEL)}
+                />
+                <SorteringHeader
+                    sortering={Sorteringsfelt.BOSTED_SIST_OPPDATERT}
+                    onClick={sorteringOnClick}
+                    rekkefolge={sorteringsrekkefolge}
+                    erValgt={sorteringsfelt === Sorteringsfelt.BOSTED_SIST_OPPDATERT}
+                    tekst="Geografisk sist oppdatert"
```

Comment:
forslag for bedre tekst?

Reply:
Hva betyr feltet? Det siste registrerte bostedet en person har hatt? 🤔 Hvis det er det, kan det jo kanskje hete "Siste registrerte bosted" eller bare "Bosted", hvis brukeren egentlig ikke har noe forhold til at det er det sist oppdaterte. Drago, if my norwegian is confusing let me know :D

Reply:
Hei Oda, sometimes its confusing, I will tell you when I cant follow. Det feltet er bare en dato, når bosted er oppdatert. Jeg endret tekst til: _Bosted sist oppdatert._..kanskje det er litt bedre enn _Geografisk sist oppdatert_.